tests/rptest/clients/kafka_cli_tools.py

```diff
@@ -105,6 +105,13 @@ def fix_key(key):
         configs = {kv[0]: maybe_int(kv[0], kv[1]) for kv in configs.items()}
         return TopicSpec(name=topic, **configs)
 
+    def describe_broker_config(self):
```

Comment:
hot. 

do we need a ticket to track the dynamic props?

Reply:
https://github.com/vectorizedio/redpanda/issues/734